test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/CustomSearchParamTests.cs

```diff
@@ -92,6 +92,67 @@ public async Task GivenANewSearchParam_WhenReindexingComplete_ThenResourcesSearc
             await DeleteSearchParameterAndVerify(searchParamPosted.Resource);
         }
 
+        [Fact]
+        public async Task GivenASearchParam_WhenUpdatingParam_ThenResourcesIndexedWithUpdatedParam()
+        {
+            var patientName = Guid.NewGuid().ToString().ComputeHash().Substring(28).ToLower();
```

Comment:
>Guid.NewGuid().ToString().ComputeHash().Substring(28).ToLower() [](start = 30, length = 63)

super nit:
Take guid -> convert it to hash ->take substring -> cast to lower case.
Why not just `Take guid`?

Reply:
:) It is true.  I simply copied this from elsewhere, and didn't put much thought into it.